.github/workflows/build-and-test.yaml

```diff
@@ -43,6 +43,13 @@ jobs:
 
   release-tarballs:
     runs-on: ubuntu-latest
+    needs:
+      - test-postgres
+      - test-mysql
+      - test-cockroach
+      - test-cassandra
+      - test-sqlite
+      - test-rqlite
```

Comment:
```suggestion
```

While we'd want this to be serialized for release we don't need that for testing (better to run tests in parallel and fail faster)